### PR TITLE
Define request collapsing

### LINKED ISSUE #475: Request collapsing
We should really mention this, as it's common and can cause misunderstanding. I also am likely to need to reference it in #474.

### PR DESCRIPTION
**Pull request by mnot**
Fixes #475

### CODE REVIEW

File: draft-ietf-httpbis-cache-latest.xml [hunk 1]
```diff
@@ -531,6 +531,15 @@
    Also, note that unsafe requests might invalidate already-stored responses;
    see <xref target="invalidation" />.
 </t>
+<t>
+   A response that is stored or can be stored can be used to satisfy multiple
+   requests, provided that it is allowed to reuse that response for the requests
```
Comment (reschke) [inline draft-ietf-httpbis-cache-latest.xml:536]: Maybe "A response that is stored or storable can be used to satisfy multiple..." ?

**Review by reschke — CHANGES_REQUESTED** [review]

File: draft-ietf-httpbis-cache-latest.xml [hunk 2]
```diff
@@ -531,6 +531,15 @@
    Also, note that unsafe requests might invalidate already-stored responses;
    see <xref target="invalidation" />.
 </t>
+<t>
+   A response that is stored or can be stored can be used to satisfy multiple
```
Comment (mnot) [inline draft-ietf-httpbis-cache-latest.xml:535]:
```suggestion
   A response that is stored or storable can be used to satisfy multiple
```

**Review by reschke — APPROVED** [review]

**Review by reschke — COMMENTED** [review]
Maybe add a minimal explanation of "collapse"?

File: draft-ietf-httpbis-cache-latest.xml [hunk 3]
```diff
@@ -531,6 +531,15 @@
    Also, note that unsafe requests might invalidate already-stored responses;
    see <xref target="invalidation" />.
 </t>
+<t>
+   A response that is stored or storable can be used to satisfy multiple
+   requests, provided that it is allowed to reuse that response for the requests
+   in question. This enables caches to "collapse" multiple requests into a single forward
```
Comment (mnot) [inline draft-ietf-httpbis-cache-latest.xml:537]: add a parenthetical more fully describing what's happening

**Review by royfielding — APPROVED** [review]

File: draft-ietf-httpbis-cache-latest.xml (hunk 3 above)
Comment (mnot) [reply draft-ietf-httpbis-cache-latest.xml:537]: Looking at this again, "multiple requests into a single forward request" says all that needs to be, I think. I will change "requests" to "incoming requests" to clarify.

Comment (mnot) [reply draft-ietf-httpbis-cache-latest.xml:537]:
```suggestion
   in question. This enables caches to "collapse" multiple incoming requests into a single forward
```

**Pull request merged**